learningresources/models.py

```diff
@@ -88,5 +92,20 @@ def save(self, *args, **kwargs):
                 slug = "{0}{1}".format(slugify(self.name), count)
                 count += 1
             self.slug = slug
+            is_create = True
```

Comment:
You are right.
And with this comment you made me realize that the update of the group name is missing in case the name has changed